src/test/java/nextstep/subway/acceptance/SectionAcceptanceTest.java

```diff
@@ -0,0 +1,144 @@
+package nextstep.subway.acceptance;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import java.util.Arrays;
+import java.util.List;
+import nextstep.subway.applicaion.dto.LineResponse;
+import nextstep.subway.applicaion.dto.StationResponse;
+import nextstep.subway.common.exception.ErrorMessage;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.jdbc.Sql;
+
+@Sql("/subway.sql")
+@DisplayName("지하철역 구간 관련 기능")
+public class SectionAcceptanceTest extends BasicAcceptanceTest {
+
+  private static final String donongStation = "도농역";
+  private static final String gooriStation = "구리역";
+  private static final String ducksoStation = "덕소역";
+
+  private static final String firstLine = "1호선";
+
+  private static final String blue = "blue";
+
+  private static final int DISTANCE = 10;
+
+  /**
+   * Given 지하철 구간을 생성하면
+   * When 지하철 구간이 생성된다.
+   * Then 지하철 노선 목록 조회 시 수정된 노선을 찾을 수 있다
```

Comment:
```suggestion
   * Given 지하철 구간을 생성하고
   * When 지하철 노선 목록 조회 시
   * Then 지하철 노선에 등록한 구간 정보를 확인할 수 있다.
```
현재 when절에서 given절의 설명이 들어가고 있는 것 같네요!